Add many new definitions from previous surveys, add terms for elicitation and discussion

davidmanheim opened the pull request:
Part 1 of adding definitions previously used. Where appropriate, noting that different terms are equivalent or nearly so, and noting where identical terms are used in incompatible ways by different previous surveys.

davidmanheim committed:
Add many new definitions from previous surveys

Part 1 of adding definitions previously used. Where appropriate, noting that different terms are equivalent or nearly so, and noting where identical terms are used in incompatible ways by different previous surveys.

lagerros commented:
Hmm... Is there any particular reason you want to include all these?

With the first definitions in the dictionary, I tried keeping all terms defined to an excellent and usable standard, and I feel worried about filling it up with less precise terms simply because they've been used elsewhere in practice.

davidmanheim commented:
> Hmm... Is there any particular reason you want to include all these?

Maybe we should split between "previous uses" and "recommended," but for most of the examples, 

1) I think it's important for people to see what others might think / mean when they see the term, so they are careful when wording questions.

2) I think it's important to be able to note when the dictionary isn't compatible with the previous literature / predictions.

>
> With the first definitions in the dictionary, I tried keeping all terms defined to an excellent and usable standard, and I feel worried about filling it up with less precise terms simply because they've been used elsewhere in practice.

That makes sense - I'm unsure if there's a good way to both include secondary definitions, and still de-emphasize them. Can we tag things as "deprecated" or "non-recommended" (Note: I think we should do this for many terms in the dictionary as well. People need to stop asking "when do you think there will be AGI," since sung the term like this makes it a very unclear / disputed question.)

lagerros commented:
I agree with you those uses are important. It seems like we should do some "separation of concerns" and optimise for different uses separately. 

By analogy, there's a difference between:

A) a dictionary
B) a thesaurus
C) an etymology dictionary

I tried to make the current thing be A), and you seem to be suggesting that B) and C) are also useful to your work. Since you're currently the main person using the Dictionary, I'm happy to go with your judgement there; but want to do so in a way which won't make it hard to optimise for A) later. 

> Can we tag things as "deprecated" or "non-recommended"

The theme we're using actually makes this really easy, just use the syntax as shown here: https://pmarsceill.github.io/just-the-docs/docs/ui-components/labels/

This might be the best solution for now, but I might want us to separate the dictionary into two documents later (instead of currently having it all in one document).

davidmanheim committed:
Use tags, and include more unclear terms

Expand dictionary to include terms likely to be referenced when predicting about AI safety approaches. 
(These will presumably not be used directly for forecasts, but may be used to forecasts about later human judgement. Example: In 2030, what is the probability that a panel of AI safety experts will unanimously agree that "Value Learning" is a solved problem?)
